basic_ingest.install

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * @file
+ * Update functions for the Basic Ingest module.
+ */
+
+/**
+ * Implements hook_update_N().
```

Comment:
Update hooks summary should just be descriptive, as in, no "Implements hook_yadda()": https://www.drupal.org/docs/develop/standards/php/api-documentation-and-comment-standards#updateN